src/main/java/com/purbon/kafka/topology/actions/topics/SyncTopicAction.java

```diff
@@ -46,6 +47,15 @@ public void run() throws IOException {
     syncTopic(topic, fullTopicName, listOfTopics);
   }
 
+  @Override
+  public void dryRun() throws IOException {
```

Comment:
is _dryRun_ a good name for this function? from one side is what we want to run during dry run, but as well is doing validations ...., what do you think? should we have two functions?

Reply:
Good point. I've called the method **Action**._dryRun_() because the other method is called **Action**._run_(). 
There are many different **BaseActions**; on topcis, accounts, and all sorts of bindings. 
Whatever - if anything - needs to happen for any of these **Actions** I don't know. The **SyncTopicAction** now only executes a schema compatibility check, but things might evolve over time when expanding this functionality.

If all we ever do is validate, we could definitely call it **Action**._validate_(). 

Disclaimer: naming things is not my strongest point, and I guess both are fine. Things can always be changed :)